### PR TITLE
Frontend bursteinz

### PR DESCRIPTION
**Pull request by zevburst**
Fixed bullet point formatting
Added “Guidelines” section

### CODE REVIEW

**Comment by zevburst** [comment]
I reverted some commits, so the reverts are shown as additional commits. I didn't realize that all my commits are added to this pull request, so the title isn't fully descriptive. My bad. See the commits for ask me or more details.

File: app/scss/components/_grid.scss [hunk 1]
```diff
@@ -166,7 +166,14 @@
   @include flex(0, 0, 90%);
   max-width: 90%;
 }
-
+.left {
```
Comment (kisphp) [inline app/scss/components/_grid.scss]: I would rename these two clases into something different. And you might not need them at all. You can use col-sm-4 and col-sm-8. Please have a look on bootstrap grid

Comment (kisphp) [inline app/scss/components/_grid.scss]: Wait. We're not using bootstrap for this ?

**Comment by kisphp** [comment]
👍

File: app/scss/components/_grid.scss (hunk 1 above)
Comment (josephrexme) [inline app/scss/components/_grid.scss]: You don't need them at all. I remember telling you our grid classes are just like that of ionic at http://ionicframework.com so you could have `.col-40` and `.col-60` or `.col-40` and `.col`. With the benefit of our own grids automatically adjusting to the space left and given `.col` a 60% width. I explained all this to you. This is what makes our grid way better than bootstrap grid. Don't edit this grid partial at all

File: public/styleguides.html [hunk 2]
```diff
@@ -1,292 +0,0 @@
-<!doctype html>
-<html class="sg">
-
-<head>
-  <meta charset="utf-8">
```
Comment (josephrexme) [inline public/styleguides.html:5]: Why did you delete styleguides.html?

File: app/scss/components/_grid.scss (hunk 1 above)
Comment (josephrexme) [inline app/scss/components/_grid.scss]: @kisphp Yes we aren't using Bootstrap because bootstrap is unnecessary bloat and that's why there was styleguides.html (Can be seen at public/styleguides.html) to show everyone our own components and how they can be used

Comment (josephrexme) [inline app/scss/components/_grid.scss]: It's like everyone contributing is just not reading through the README and CONTRIBUTING guide and that's too bad

Comment (kisphp) [inline app/scss/components/_grid.scss]: @bl4ckdu5t you're right. I will read them. I just looked in the gulpfile and I saw that bootstrap is not used any more. I'll have a deeper look in the repository next week when I'll have a laptop again :D

**Comment by kisphp** [comment]
I see that first commit is a merge from master. this should not happen to anyone.

for everybody please use the following commands in terminal when you create new branches.

```
git checkout master # move to master branch if you are not there

git checkout -b my-new-branch

git add . # add your changed files

git commit -m 'what I did in those files'

git push origin my-new-branch
```

Don't use GUIs for this .

**Comment by kisphp** [comment]
:+1:

**Comment by kisphp** [comment]
@bl4ckdu5t how long will take to have some js tests ?
